### DISCUSSION TITLE
Utente sia docente che genitore

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#561] [Richieste di aiuto] [answered]
Buongiorno, per l'anno scolastico 2023/2024 sarà presente un docente che lavorerà nella nostra scuola. Il fatto è che suo figlio è uno dei nostri alunni. In questo caso, come funziona l'accesso al registro? Non credo sia opportuno creare un doppio account per il docente e il genitore. È possibile accedere e scegliere se entrare come docente o genitore?

**Comment by trinko** [comment, marked as answer]
Funziona in questo modo: 
- dal punto di vista dell'amministrazione è necessario configurare due utenti, uno per il docente e poi ovviamente quello del genitore associato all'alunno; è fondamentale che i due utenti riportino gli stessi identici dati che identificano la stessa persona: NOME, COGNOME e CODICE FISCALE;
- per l'utente, al momento del login come docente, il sistema trova l'altro utente genitore (identificato da NOME/COGNME/CODICE-FISCALE) e propone una pagina di scelta del profilo desiderato.

In altre parole, accedendo come docente può scegliere il profilo docente o il profilo genitore. Tornando successivamente alla _Home_ si potrà nuovamente cambiare profilo.
Attenzione che se l'utente effettua il login come GENITORE non gli sarà possibile accedere al profilo DOCENTE, questo perché per genitori  si ha una password con un livello di sicurezza inferiore (8 caratteri anziché 10).

Lo stesso meccanismo è utilizzato per genitori che hanno 2 o più figli nella scuola: in questo modo possono usare sempre un solo utente e cambiare profilo successivamente.
